scripts/gen-aliasarr.template.sh

```
@@ -599,25 +612,27 @@ _arr_wait_for_container_health() {
 }
 
 _arr_stack_restart_services() {
-  local -a order=()
+  local order=""
```

Shellcheck (0.11.0)

[warning] order appears unused. Verify use (or export if used externally).

(SC2034)